.github/workflows/create-package.yml

```diff
@@ -0,0 +1,40 @@
+﻿name: Create and publish package
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  package-release:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        configuration: [BelowZero, Subnautica]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get artifact from build
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: build.yml
+        name: SCHIZO-${{ matrix.configuration }}.dll
+        path: Install/${{ matrix.configuration }}/BepInEx/plugins/SCHIZO/
+
+    - name: Create release archive
+      run: |
+          cd Install/${{ matrix.configuration }}
+          zip -r9 ../mod-and-deps-${{ matrix.configuration }}.zip *
+          cd BepInEx/plugins
+          zip -r9 ../../../mod-only-${{ matrix.configuration }}.zip SCHIZO
+          cd ../../../Common
+          zip -r9 ../mod-and-deps-${{ matrix.configuration }}.zip *
+
+    - name: Update the release
+      uses: softprops/action-gh-release@v1
+      with:
+        files: |
+          Install/mod-and-deps-${{ matrix.configuration }}.zip
+          Install/mod-only-${{ matrix.configuration }}.zip

```

Comment:
```suggestion
          Install/SCHIZO-${{ matrix.configuration }}-with-dependencies.zip
          Install/SCHIZO-${{ matrix.configuration }}.dll
```

Reply:
Noted that you want just the `.dll`. Will change this and the above comment (if applicable) with one commit once you reply to it